Source/UnrealGT/Public/Generators/Image/GTSceneCaptureComponent2D.h

```diff
@@ -9,14 +9,15 @@
 
 #include "GTImage.h"
 #include "GTObjectFilter.h"
+#include "GTRandomColorGenerator.h"
 
 #include "GTSceneCaptureComponent2D.generated.h"
 
 class UTexture2D;
 class UMaterial;
 class UMaterialInstanceDynamic;
 class UGTImageGeneratorBase;
-
+class RandomColorGenerator;
```

Comment:
This forward declaration seems to be redundant/incorrect, since GTColorGenerator is already included above.

Reply:
Thanks, yes, forgot to delete.